app/wama.web/Controllers/UserToolController.cs

```diff
@@ -239,7 +239,13 @@ public IActionResult AddCertification(string memberId)
             ViewBag.MemberId = memberId;
             return View($"{Constants.ADMIN_CONSOLE_USER_TOOL_DIRECTORY}/ViewCertification.cshtml", Certifications);
         }
-
+        public async Task<IActionResult> ViewOneCertification(string memberId) //need to straight idea first
```

Comment:
- [ ] What about changing this to, ViewCertification (it still implies you're looking at one certification)?
- [ ] Also, for the parameter, shouldn't be **_certificationID_**?
- [ ] I'm also thinking that the parameter should be taking the **_memberID_** as well (_**since we want to map it down to a certification, that belongs to a user**_) (**_this is optional_**)?